samples/server/petstore/kotlin-server-modelMutable/src/main/kotlin/org/openapitools/server/Paths.kt

```diff
@@ -21,43 +21,43 @@ object Paths {
      * 
      * @param body Pet object that needs to be added to the store 
      */
-    @Serializable @Resource("/pet") class addPet(val body: Pet)
+    @Resource("/pet") class addPet()
 
     /**
      * Deletes a pet
      * 
      * @param petId Pet id to delete 
      * @param apiKey  (optional)
      */
-    @Serializable @Resource("/pet/{petId}") class deletePet(val petId: kotlin.Long, val apiKey: kotlin.String? = null)
+    @Resource("/pet/{petId}") class deletePet(val petId: kotlin.Long)
 
     /**
      * Finds Pets by status
      * Multiple status values can be provided with comma separated strings
      * @param status Status values that need to be considered for filter 
      */
-    @Serializable @Resource("/pet/findByStatus") class findPetsByStatus(val status: kotlin.collections.MutableList<kotlin.String>)
+    @Resource("/pet/findByStatus") class findPetsByStatus()
```

Comment:
what's the reason to remove the parameters (e.g. query) from the method signature?

looks to me this is also a breaking change so ideally i would like to avoid this if possible if we want to include it in the upcoming "minor" release (which allows breaking changes with fallback)

Reply:
Fixed